cmd/clusterctl/client/repository/components.go

```diff
@@ -42,16 +42,12 @@ const (
 	clusterRoleKind                    = "ClusterRole"
 	clusterRoleBindingKind             = "ClusterRoleBinding"
 	roleBindingKind                    = "RoleBinding"
+	certificateKind                    = "Certificate"
 	mutatingWebhookConfigurationKind   = "MutatingWebhookConfiguration"
 	validatingWebhookConfigurationKind = "ValidatingWebhookConfiguration"
 	customResourceDefinitionKind       = "CustomResourceDefinition"
 )
 
-const (
```

Comment:
Unused and doesn't make sense anymore. I suppose it's fine to delete it? (I can add it again if necessary)